README.aspnetcore.md

```diff
@@ -10,23 +10,23 @@ These images contain the runtime only. Use [`microsoft/aspnetcore-build`](https:
 
 # Supported Linux amd64 tags
 
-- [`1.0.9-jessie`, `1.0.9`, `1.0` (*1.0/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/jessie/runtime/Dockerfile)
+- [`1.0.10-jessie`, `1.0.10`, `1.0` (*1.0/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/jessie/runtime/Dockerfile)
 - [`1.1.6-jessie`, `1.1.6`, `1.1`, `1` (*1.1/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/runtime/Dockerfile)
```

Comment:
I was expecting to see a version rev for the two 1.1.* images.

Reply:
Good catch, yes this should bump.